dissect/target/helpers/configutil.py

```diff
@@ -240,6 +240,35 @@ def parse_file(self, fh: TextIO) -> None:
         self.parsed_data = information_dict
 
 
+class Csvish(Default):
+    """Parses CSV-ish config files (does not confirm to CSV standard!)"""
+
+    def __init__(self, *args, **kwargs) -> None:
+        self.fields = kwargs.pop("fields")
+        self.num_fields = len(self.fields)
+        self.maxsplit = self.num_fields - 1
```

Comment:
Fields should be mandatory, otherwise your config becomes unsearchable, you must know what you are mapping. It makes no sense otherwise as far as I can see. Column headers are also not a thing in practice.